src/proxy/proxy.factory.ts

```diff
@@ -41,13 +63,15 @@ export const proxyFactory = (params: ProxyParams) => async (req: IncomingMessage
       const response = await makeRequest(url, options)
       const originalBody = originalRequest.body
       return {
+        url,
         originalBody,
         response
       }
     })
-    .then(async ({ originalBody, response }) => {
+    .then(async ({ url, originalBody, response }) => {
       const { headers, statusCode, statusMessage } = response
       const data = await dataMapper(response.data, originalBody)
+      await logRequest(url, statusCode, statusMessage, originalBody, response.data, data)
```

Comment:
This will block every request. Would be better to inject verbosity value or (at least) remove `await`